extensions/functions_arithmetic.yaml

```diff
@@ -1368,6 +1368,43 @@ aggregate_functions:
         nullability: DECLARED_OUTPUT
         ordered: true
         return: LIST?<any>
+  
+  - name: "any_value"
+    description: >
+      Selects an arbitrary value from a group of values.
+
+      If the input is empty, the function returns null.
+    impls:
+      - args:
+          - name: x
+            value: i8
+        nullability: DECLARED_OUTPUT
+        return: i8?
+      - args:
+          - name: x
+            value: i16
+        nullability: DECLARED_OUTPUT
+        return: i16?
+      - args:
+          - name: x
+            value: i32
+        nullability: DECLARED_OUTPUT
+        return: i32?
+      - args:
+          - name: x
+            value: i64
+        nullability: DECLARED_OUTPUT
+        return: i64?
+      - args:
+          - name: x
+            value: fp32
+        nullability: DECLARED_OUTPUT
+        return: fp32?
+      - args:
+          - name: x
+            value: fp64
+        nullability: DECLARED_OUTPUT
+        return: fp64?
```

Comment:
This is completely covered by the `f(any) -> any?` variant, and it isn't really an arithmetic function to begin with. So why is this still here?